src/org/openj9/envInfo/JavaInfo.java

```diff
@@ -25,6 +29,10 @@ public String getSPEC(String javaImplInfo) {
         System.out.println("System.getProperty('java.fullversion')=" + fullversion + "\n");
         String spec = "";
         if (osName.contains("linux")) {
+            Path alpine = Paths.get("/etc/alpine-release");
+			if (Files.exists(alpine)) {
+				spec = "linux_x86-64";
+			}
```

Comment:
- The logic is not correct. If exists(alpine), the spec is `linux_x86-64`? 
Please see example https://github.com/adoptium/aqa-tests/blob/master/jck/jtrunner/JavaTestRunner.java#L1240
- spec will be appended with arch and bit information in the logic below. We only set os here (i.e., `alpine-linux`), not hardcode the whole spec.
- please run a Grinder after the code change.

Reply:
Thanks for the correction. I'd like to ask should the `osName.equals("linux")` be equivalent to `osName.contains("linux")` here? 

Also, may I ask what's the reason for changing the spec to `alpine-linux_x86-64`?

Reply:
>I'd like to ask should the osName.equals("linux") be equivalent to osName.contains("linux") here?

Not sure if osName contains extra strings other than linux, I guess contains is better than equals.

> Also, may I ask what's the reason for changing the spec to alpine-linux_x86-64?

You will also need to update ottawa.csv file, this file controls which spec runs on what platform for what JVM option. Add a column of  alpine-linux_x86-64 in it. We use different spec to run or skip the test based on ottawa.csv.

Two comments on your current change:
- use space instead of tab for indentation so it follows the existing format
- you need to have the `spec="linux"` in else condition